public/js/machine.js

```diff
@@ -1 +1,37 @@
-$('#machine-table').tablesort();
+// $('#machine-table').tablesort();
+function generateCalendar(calendar) {
+  let eventDates = [];
+  const bucket = 0;
+  const today = 1;
+  const backup = 2;
+  const id = 3;
+  const dates = 4;
+  const year = 2021;
```

Comment:
Instead of hard-coding the year, get the current year by constructing a new `Date` object and using the `getFullYear` method.